frontend/src/metabase/components/MetadataInfo/DimensionInfo/DimensionInfo.info.js

```diff
@@ -1,6 +1,10 @@
 import React from "react";
 
-import { PRODUCTS, ORDERS, metadata } from "__support__/sample_dataset_fixture";
+import {
+  PRODUCTS,
+  ORDERS,
+  metadata,
+} from "__support__/sample_database_fixture";
```

Comment:
this change is curious to me. Did we change the prettier settings? Or is the one extra character enough to make the linter spill it over multiple lines?

Reply:
I think it's just the extra character causing it to hit the line length, yeah